specs/environments/6.0_domain/systemtest.env

```diff
@@ -0,0 +1,13 @@
+JBOSS_CONTROLLER_URL=jboss:9999
+JBOSS_USERNAME=admin
+JBOSS_PASSWORD=changeme
+JBOSS_CLI_PATH=/opt/jboss/bin/jboss-cli.sh
+JBOSS_LOG_LEVEL=DEBUG
+JBOSS_MODE=domain
+RESOURCE_HOSTNAME=jboss
+RESOURCE_PORT=7808
+
```

Comment:
most likely we will need to add here one more env variable indicating EAP version. It may be used for detecting e.t. whether deploy --url feature will fail or will succeed